frontend/src/js/external-forms/form-concept-group/FormConceptGroup.tsx

```diff
@@ -112,6 +112,17 @@ const FormConceptGroup = (props: Props) => {
   const newValue = props.newValue;
   const defaults = props.defaults || {};
 
+  // indicator if it should be scrolled down back to the dropZone
+  const [scrollToDropzone, setScrollToDropzone] = useState<boolean>(false);
+  const dropzoneRef = createRef<HTMLDivElement>();
```

Comment:
`createRef` creates a ref every time it's called, so in this case, it will create a new ref on every rerender of FormConceptGroup. Is there a reason you'd go for `createRef` instead of `useRef`? Otherwise, let's use `useRef` please

Reply:
No particular reason, changed it to `useRef`